src/main/resources/org/jenkinsci/plugins/workflow/steps/MailStep/help.html

```diff
@@ -1,3 +1,6 @@
 <div>
     Simple step for sending email.
+
+    <p>Use the <a href="https://www.jenkins.io/redirect/pipeline-snippet-generator">Pipeline Snippet Generator</a> to generate a sample pipeline script for the mail step.</p>
```

Comment:
I meant the same for all other steps, like:
```suggestion
    <p>Use the <a href="https://www.jenkins.io/redirect/pipeline-snippet-generator">Pipeline Snippet Generator</a> to generate a sample pipeline script for the <code>mail</code> step.</p>
```